linter/src/main/java/linter/App.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package linter;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Scanner;
+
+public class App {
+    public String getGreeting() {
+        return "Hello world.";
+    }
+
+    public static void main(String[] args) {
```

Comment:
these are a good start to test as you're coding, but these so not count as tests